.github/workflows/generate-summary.yaml

```diff
@@ -415,13 +415,25 @@ jobs:
           title: '${{ inputs.issue_hash_prefix }} ${{ inputs.gcchash }}'
           labels: ${{ steps.issue-labels.outputs.issue_labels }}
 
+      - name: Build additional info link
+        id: issue-info
+        run: |
+          sleep 1
+          curl -L -H "Accept: application/vnd.github+json" -H "Authorization: Token ${{ secrets.GITHUB_TOKEN }}" -H "X-GitHub-Api-Version: 2022-11-28" https://api.github.com/repos/patrick-rivos/gcc-postcommit-ci/issues/${{ steps.create-issue.outputs.issue-number }} | jq '.id' > issue_body_comment_id.txt
+          export ISSUE_BODY_COMMENT=$(cat issue_body_comment_id.txt)
+          export LINK=https://github.com/patrick-rivos/gcc-postcommit-ci/issues/${{ steps.create-issue.outputs.issue-number }}#issue-$(cat issue_body_comment_id.txt)
+          echo $LINK
+          echo "additional=$LINK" >> $GITHUB_OUTPUT
```

Comment:
```suggestion
          echo "base_comment_link=$LINK" >> $GITHUB_OUTPUT
```

Reply:
If this change is made, should the name of the step also change to `Build base comment link`?